### ISSUE TITLE
Dupes the active item in the hand

### ISSUE THREAD

**Issue by raspopov** [ricksouth/serilum-mc-mods#1035]
## **Information**
Minecraft version: 1.18.2
Modloader: Fabric
Fabric loader version: 0.50.0
Environment: Multiplayer

Mod name: Your Items Are Safe (Fabric)
Mod version: 1.1 (02.02.2022)

## **Description**
Dupes the active item in the hand. 😣😣

youritemsaresafe-fabric.json:
```
{
	// When enabled and a player dies without any items in their inventory, no chest or armor stand is generated.
	"mustHaveItemsInInventoryForCreation": false,
	// If a player head should be added to the armor stand. If a helmet is worn, this will be placed into the chest.
	"addPlayerHeadToArmorStand": false,
	// Whether an armor stand should be created on death. If disabled, the player's gear will be placed inside the chest.
	"createArmorStand": false,
	// Whether a sign should be placed on the chest with the name of the player who died there.
	"createSignWithPlayerName": true,
	// Whether materials are needed for the chest which spawns on death. This can be the actual chest or the costs in raw materials.
	"needChestMaterials": false,
	// Whether materials are needed for the armor stand to spawn on death. This can be the actual armor stand or the costs in raw materials.
	"needArmorStandMaterials": false,
	// Whether materials are needed for the creation of the sign when 'createSignWithPlayerName' is enabled.
	"needSignMaterials": false,
	// Only relevant if 'needChestAndArmorStandMaterials' is enabled. An armor stand needs 1 stone slab to be created, but I think it's alright to ignore that requirement. If enabled, no stone is needed in the inventory on death.
	"ignoreStoneMaterialNeed": true,
	// If a message should be sent if the chest or armor stand can't be created due to missing materials.
	"sendMessageOnCreationFailure": true,
	// If a message should be sent on successful creation of the chest(s) and armor stand.
	"sendMessageOnCreationSuccess": true,
	// The message sent on creation failure with 'sendMessageOnCreationFailure' enabled. Possible replacement values: %plankamount%, %stoneamount%.
	"creationFailureMessage": "Your items are not safe due to having insufficient materials. Missing: %plankamount% planks.",
	// The message sent on creation success with 'sendMessageOnCreationSuccess' enabled.
	"creationSuccessMessage": "Your items are safe at your death location."
}
```

**Comment by KX5-jeron** [comment]
Came here to report this. Any item we have in our main hand gets duped and placed in the chest upon death.

**Comment by ghost** [comment]
This is absolutely game breaking. Needed to remove the mod from our server. Thousands of duped items. Very frustrating. @ricksouth is there an eta on a fix for this? I can't find any other fabric vanilla compatible death chest mod...

**Comment by ricksouth** [comment]
I'm very sorry for any inconvenience caused by this. It was an oversight on my end, and only happened when the armor stand creation was set to false. I didn't test that thoroughly enough.

This has been fixed in version 2.0 of Your Items Are Safe, along with some other bug fixes. I appreciate you giving the mod a try and opening the issue/commenting!

https://www.curseforge.com/minecraft/mc-mods/your-items-are-safe
https://www.curseforge.com/minecraft/mc-mods/your-items-are-safe-fabric
